src/pages/SettingsPage.js

```diff
@@ -35,15 +38,15 @@ export default function SettingsPage({ padSettings, setPadSettings }) {
   function sampleChange(e){
     setSelectedSample(e.target.value)
   }
-
   function savePadClick() {
-    const removedPad = padSettings.splice(selectedPad, 1);
+    const removedPad = allPads.splice(selectedPad, 1);
```

Comment:
`.splice()` mutates your `allPads` state, you need to make a copy beforehand [have a look here on .splice()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/splice)